ui/app/views/demo.html

```diff
@@ -64,7 +64,7 @@ <h2 class="col-md-12">{{model.demo.name}} <a class="btn btn-primary btn-xs" ng-h
         <autocomplete ng-model="model.additionalBug.assignee" data="model.users" click-activation="true" ></autocomplete>
       </fieldset>
       <div class="col-md-12">
-        <a class="col-md-1" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
+        <a class="btn btn-default btn-xs" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
```

Comment:
The Cancel was an 'a' so I left it and just added the btn styles. The classes are designed for a, button, or input.